Source/cmake/OptionsGTK.cmake

```diff
@@ -85,10 +84,6 @@ WEBKIT_OPTION_DEPEND(ENABLE_DOCUMENTATION ENABLE_INTROSPECTION)
 WEBKIT_OPTION_DEPEND(ENABLE_3D_TRANSFORMS USE_OPENGL_OR_ES)
 WEBKIT_OPTION_DEPEND(ENABLE_ASYNC_SCROLLING USE_OPENGL_OR_ES)
 WEBKIT_OPTION_DEPEND(ENABLE_GLES2 USE_OPENGL_OR_ES)
-WEBKIT_OPTION_DEPEND(ENABLE_WEBGL USE_OPENGL_OR_ES)
```

Comment:
GTK port has `USE_OPENGL_OR_ES` build switch to disable OpenGL usage for some minor operating systems which has a problem for OpenGL.
I think they can't use WebGL even with ANGLE, `WEBKIT_OPTION_DEPEND(ENABLE_WEBGL USE_OPENGL_OR_ES)` shouldn't be removed.

Reply:
I understand this option has been broken for a while, though? Maybe it's time to just remove it?

But yeah, as long as the option still exists, the `WEBKIT_OPTION_DEPEND` should indeed not be removed. Good catch.

Reply:
I think someone fixed the problem very recently. https://github.com/WebKit/WebKit/pull/4035#discussion_r964183001

Reply:
OK, this line should remain then.

Reply:
It should remain. Same DEPEND should be added for WEBGL2, until it gets cleaned up (maybe by combining `ENABLE_WEBGL2` into `ENABLE_WEBGL`).

Reply:
thanks